typescript/looker/rtl/constants.ts

```diff
@@ -22,8 +22,8 @@
  * THE SOFTWARE.
  */
 
-export const lookerVersion = '6.24'
-export const apiVersion = '3.1'
+export const lookerVersion = '6.25'
+export const apiVersion = '4.0'
```

Comment:
intentional?

Reply:
whoops, missed reverting that. Good catch.